config.template.yml

```
@@ -16,6 +16,8 @@ server:
   read_buffer_size: 4096
   # Write buffer size configures the http server's maximum outgoing response size in bytes.
   write_buffer_size: 4096
+  # Set the path Authelia listens on, must be alphanumeric chars.
```

To me it's not clear that the path must be a single level path. Let's improve the comment
```suggestion
  # Set the single level path Authelia listens on, must be alphanumeric chars and slash is forbidden.